sklearn/utils/validation.py

```diff
@@ -212,6 +213,26 @@ def check_consistent_length(*arrays):
                          " samples: %r" % [int(l) for l in lengths])
 
 
+def _convert_iterable(iterable):
```

Comment:
The goal of this function is to convert iterable containers to a similar indexable container (if not iterable by default). The following name better reflect this:

```suggestion
def _make_indexable(iterable):
```

Reply:
I cannot accept the suggestions since it is not my PR. I will add this by hand.